source/includes/_send_whatsapp.md

```diff
@@ -0,0 +1,244 @@
+  }
+});
+
+xhr.open("POST", "%7B%7Bbase_url%7D%7D/api/whatsapp/send-link");
+xhr.setRequestHeader("Content-Type", "application/json");
+xhr.setRequestHeader("Accept", "application/json");
+xhr.setRequestHeader("x-oy-username", "{{username}}");
+xhr.setRequestHeader("x-api-key", "{{api-key}}");
+
+xhr.send(data);
+```
+
+```php
+<?php
+require_once 'HTTP/Request2.php';
+$request = new HTTP_Request2();
+$request->setUrl('{{base_url}}/api/whatsapp/send-link');
+$request->setMethod(HTTP_Request2::METHOD_POST);
+$request->setConfig(array(
+  'follow_redirects' => TRUE
+));
+$request->setHeader(array(
+  'Content-Type' => 'application/json',
+  'Accept' => 'application/json',
+  'x-oy-username' => '{{username}}',
+  'x-api-key' => '{{api-key}}'
+));
+$request->setBody('{\n	"partner_tx_id": "partnerTxId",\n	"whatsapp_number": "621234567890"\n}');
+try {
+  $response = $request->send();
+  if ($response->getStatus() == 200) {
+    echo $response->getBody();
+  }
+  else {
+    echo 'Unexpected HTTP status: ' . $response->getStatus() . ' ' .
+    $response->getReasonPhrase();
+  }
+}
+catch(HTTP_Request2_Exception $e) {
+  echo 'Error: ' . $e->getMessage();
+}
+```
+
+```python
+import http.client
+import json
+
+conn = http.client.HTTPSConnection("{{base_url}}")
+payload = json.dumps({
+  "partner_tx_id":"partnerTxId",
+  "whatsapp_number":"621234567890"
+})
+headers = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'x-oy-username': '{{username}}',
+  'x-api-key': '{{api-key}}'
+}
+conn.request("POST", "/api/whatsapp/send-link", payload, headers)
+res = conn.getresponse()
+data = res.read()
+print(data.decode("utf-8"))
+```
+
+> JSON Response
+
+```json
+{
+    "status": {
+        "code": "000",
+        "message": "Success"
+    },
+    "data": {
+        "partner_tx_id": "partnerTxId",
+        "whatsapp_number": "621234567890",
+        "success_sent": true
+    }
+}
+```
+
+### HTTPS Request
+
+**[Production]** `POST https://partner.oyindonesia.com/api/whatsapp/send-link`
+
+**[Staging]** `POST https://api-stg.oyindonesia.com/api/whatsapp/send-link`
+
+### Flow
+
+<Fill below with proper image>
+[Image: Screen Shot 2022-01-28 at 09.10.16.png]
+
+
+### Request Headers
+
+|Parameter	|Type	|Description	|
+|---	|---	|---	|
+|X-Api-Key	|String	|API Key used to connect to this particular endpoint	|
+|X-Oy-Username	|String	|Your OY! account username	|
+
+### Request Parameters
+
+|Parameter	|Type	|Required	|Default	|Description	|
+|---	|---	|---	|---	|---	|
+|partner_tx_id	|String(255)	|TRUE	|-	|ID generated by Partner for a specific transaction	|
+|whatsapp_number	|String(255)	|TRUE	|-	|The WhatsApp number of your users. (format 628xxxxxxxxxx)	|
+
+### Response Parameters
+
+|Parameter	|Type	|Description	|
+|---	|---	|---	|
+|status	|Object	|Status of response in Object `{code: <status_code>, message: <status_message>}`.	|
+|data	|Object	|The API's response data in an Object, details are in the separate table below.	|
```

Comment:
in the "Response Data Parameters table." --> possible buat dibikin hyperlink kah?